src/apps/config/leader.lua

```diff
@@ -114,28 +114,30 @@ function Leader:rpc_describe (args)
             capability = schema.get_default_capabilities() }
 end
 
-local function path_printer_for_grammar(grammar, path)
+local function path_printer_for_grammar(grammar, path, print_default)
    local getter, subgrammar = path_mod.resolver(grammar, path)
+   data.print_default(print_default or false)
```

Comment:
"print_default or false" is the same as "print_default", at least in a boolean context.